dev-client/jest/setup.ts

```diff
@@ -0,0 +1,54 @@
+import '@testing-library/jest-native';
```

Comment:
issue: add copyright notice.

Reply:
@paulschreiber I noticed only one file has this notice right now. Let's use [eslint-plugin-header](https://www.npmjs.com/package/eslint-plugin-header) to catch this and add the copyright automatically for the other files in another PR.